src/scala/org/pantsbuild/zinc/compiler/InputUtils.scala

```diff
@@ -66,16 +67,7 @@ object InputUtils {
         .withScalacOptions(scalacOptions.toArray)
         .withJavacOptions(javacOptions.toArray)
         .withOrder(compileOrder)
-    val reporter =
-      ReporterUtil.getDefault(
-        ReporterUtil.getDefaultReporterConfig()
-          .withMaximumErrors(Int.MaxValue)
-          .withUseColor(settings.consoleLog.color)
-          .withMsgFilters(settings.consoleLog.msgPredicates.toArray)
-          .withFileFilters(settings.consoleLog.filePredicates.toArray)
-          .withLogLevel(settings.consoleLog.javaLogLevel)
-          .withPositionMapper(positionMapper)
-      )
+    val reporter = ReporterUtil.getReporter(DummyLogger, ReporterManager.getDefaultReporterConfig)
```

Comment:
Do we understand what the root cause is that meant we needed to do this, so that we can be confident that there isn't other code in Zinc that will fail, or behave differently, which we may not have hit in testing yet?

Reply:
The normal logging actually works now after making a very simple substitution (see the TODO I've added in the PR description), but adds a whopping 5 seconds to startup time compared to the `DummyLogger`. But I intended to add very clear documentation on how every build or runtime failure was investigated and worked around before I mark this un-WIP.

Reply:
Do we know _why_ using the standard logger adds 5 seconds of startup time? That sounds scary...